versioneer.py

```diff
@@ -10,7 +10,7 @@
 * https://github.com/warner/python-versioneer
 * Brian Warner
 * License: Public Domain
-* Compatible With: python2.6, 2.7, 3.3, 3.4, 3.5, and pypy
+* Compatible With: python3.5+, conda, and pypy
```

Comment:
Please revert this change, since this is just a vendored copy of `versioneer`.

```suggestion
* Compatible With: python2.6, 2.7, 3.3, 3.4, 3.5, and pypy
```